### ISSUE TITLE
'Confirm' button in Label picker not working

### ISSUE THREAD

**Issue by codemanmeet** [HubTurbo/HubTurbo#1381]
<img width="422" alt="screen shot 2016-03-20 at 4 45 02 pm" src="https://cloud.githubusercontent.com/assets/4116346/13903557/500d9aa8-eebb-11e5-89bd-d0799935075f.png">

**Comment by codemanmeet** [comment]
Pressing <kbd>Enter</kbd> works though!

**Comment by damithc** [comment]
Worked for me. What exact steps did you take? May be a Mac-specific bug?

**Comment by codemanmeet** [comment]
I first right clicked and chose `Change Labels`. After that, without using any keys, I selected the label using a mouse click. The `tick` mark appeared for the label in the label picker window. I clicked on `Confirm`. The window closed but the label wasn't reflected. Here's the gif to explain the problem - 

![label-bug](https://cloud.githubusercontent.com/assets/4116346/13903711/de77b6e2-eec1-11e5-9e3a-7c3f2c1d9876.gif)

**Comment by codemanmeet** [comment]
However, once the label picker window is opened and I choose my keyboard to select one label and then press <kbd>Enter</kbd>, the label picking works fine. Here's the gif - 

![label-bug-2](https://cloud.githubusercontent.com/assets/4116346/13903732/6068e9fa-eec2-11e5-9e5f-af3792ef652a.gif)

**Comment by codemanmeet** [comment]
Could be a Mac issue!

**Comment by damithc** [comment]
I don't have the 'change label' option in my context menu :-o Is that a Mac-only thing?

**Comment by codemanmeet** [comment]
Oh, nope it's not a Mac-only thing. It should appear on a right click of an issue in the context menu that pops up. I noticed it as well. We should file another bug for that!

**Comment by damithc** [comment]
@codemanmeet w.r.t Confirm button, what happens if you use the keyboard shortcut to bring up the label picker but use the mouse after that? I tried that and seems to work.

**Comment by codemanmeet** [comment]
Nope, using the mouse to select the label after the keyboard shortcut doesn't work for me :-(
